components/support/sql/src/lib.rs

```diff
@@ -38,6 +38,33 @@ pub fn escape_string_for_pragma(s: &str) -> String {
     s.replace('\'', "''")
 }
 
+/// Default SQLite pragmas
+///
+/// Most components should just stick to these defaults.
+pub fn setup_sqlite_defaults(conn: &rusqlite::Connection) -> rusqlite::Result<()> {
```

Comment:
this appears to only be used by tests - is that intentional?

Reply:
The intention was to just use these for this example component that I'm putting together and future components.  However, we should probably eat our dogfood.  I updated suggest to start using this.  I could switch over more components too if you think that makes sense.